pkg/controller/common/nodenetwork.go

```diff
@@ -14,6 +14,7 @@ import (
 )
 
 const (
+	Namespace        = "harvester-system"
```

Comment:
why don't we get ns from https://github.com/harvester/network-controller-harvester/blob/master/main.go#L47

Reply:
It's the namespace where kube-vip deploys.

Reply:
/s/Namespace/KubeVipNamespace/